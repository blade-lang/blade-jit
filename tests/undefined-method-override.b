class A {
  @new() {
    self.value = 2
  }
}

echo new A() + new A()
