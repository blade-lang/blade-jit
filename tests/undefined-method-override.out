^^^
ArgumentError: '+' is not defined for call signature (A, A)